doc/Language/quoting.pod6

```diff
@@ -174,38 +174,26 @@ My favorite color is blue!
 =end code
 
 X<|\ (quoting)>
+
 The C<qq> form – usually written using double quotes – allows for
-interpolation of backslash sequences and variables, i.e., variables can be
-written within the string so that the content of the variable is inserted into
-the string. It is also possible to escape variables within a C<qq>-quoted
-string:
+interpolation of backslash escape sequences (like C<q:backslash>), all sigiled
+variables (like C<q:scalar:array:hash:function>), and any code inside C<{...}>
+(like C<q:closure>).
 
-=begin code :allow<B> :preamble<my $color = 'blue'>
-say B<">The B<\>$color variable contains the value '$color'B<">;
-=end code
+=head3 Interpolating variables
 
-=begin code :lang<text>
-The $color variable contains the value 'blue'
-=end code
+Inside a C<qq>-quoted string, you can use variables with a sigil to trigger
```

Comment:
This is the second time you mention variables `with a sigil`; however, if they don't have a sigil, how can they be identified as variables? Also, arguably you're talking about identifiers, not variables. Something with a sigil is clearly an identifier; no sigil, well, it could be, but not always.